### PR TITLE
Set logging memory limits

### PR DESCRIPTION
**Pull request by sdodson**
Set logging limits per Peter's analysis

### CODE REVIEW

**Comment by sdodson** [comment]
@stevekuznetsov should we move these to general.yml instead?

File: sjb/inventory/group_vars/OSEv3/logging.yml [hunk 1]
```diff
@@ -5,5 +5,16 @@ openshift_logging_es_log_appenders:
...
+openshift_logging_curator_ops_memory_limit: "32Mi"
+openshift_logging_elasticsearch_proxy_memory_limit: "32Mi"
 openshift_logging_es_memory_limit: "4Gi"
 openshift_logging_es_ops_memory_limit: "4Gi"
+openshift_logging_eventrouter_memory_limit: "64Mi
+openshift_logging_fluentd_memory_limit: "128Mi"
+openshift_logging_kibana_memory_limit: "128Mi
+openshift_logging_kibana_ops_memory_limit: "128Mi"
+openshift_logging_kibana_ops_proxy_memory_limit: "64Mi"
+openshift_logging_kibana_proxy_memory_limit: "64Mi""
```
Comment (richm) [inline sjb/inventory/group_vars/OSEv3/logging.yml]: extra trailing double quote

File: sjb/inventory/group_vars/OSEv3/logging.yml [hunk 2]
```diff
@@ -5,5 +5,16 @@ openshift_logging_es_log_appenders:
   - "console"
 openshift_logging_fluentd_journal_read_from_head: false
 openshift_logging_fluentd_audit_container_engine: true
+
+openshift_logging_curator_memory_limit: "32Mi"
+openshift_logging_curator_ops_memory_limit: "32Mi"
+openshift_logging_elasticsearch_proxy_memory_limit: "32Mi"
 openshift_logging_es_memory_limit: "4Gi"
 openshift_logging_es_ops_memory_limit: "4Gi"
+openshift_logging_eventrouter_memory_limit: "64Mi
```
Comment (richm) [inline sjb/inventory/group_vars/OSEv3/logging.yml]: missing trailing double quote

File: sjb/inventory/group_vars/OSEv3/logging.yml [hunk 3]
```diff
@@ -5,5 +5,16 @@ openshift_logging_es_log_appenders:
...
+openshift_logging_fluentd_memory_limit: "128Mi"
+openshift_logging_kibana_memory_limit: "128Mi
```
Comment (portante) [inline sjb/inventory/group_vars/OSEv3/logging.yml]: And here...

File: sjb/inventory/group_vars/OSEv3/logging.yml [hunk 4]
```diff
@@ -5,5 +5,16 @@ openshift_logging_es_log_appenders:
...
+openshift_logging_kibana_ops_memory_limit: "128Mi"
+openshift_logging_kibana_ops_proxy_memory_limit: "64Mi"
+openshift_logging_kibana_proxy_memory_limit: "64Mi""
+openshift_logging_mux_memory_limit: "256Mi
```
Comment (portante) [inline sjb/inventory/group_vars/OSEv3/logging.yml]: missing one here, too.

**Pull request merged**